packages/ListBox/test/cypress/ListBox.cypress.js

```diff
@@ -81,7 +80,7 @@ describe("ListBox single select", () => {
     it("should filter by option label", () => {
       cy.get(selectors.filterInput)
         .focus()
-        .type("spiderman");
+        .type("spi", typingDelay);
```

Comment:
This `typingDelay` slows down the test a bit, but it was getting very flakey otherwise – observing the result in Cypress I noticed the filter would contain `spman` or `deman` and things like that.  

There were a [number of threads](https://github.com/cypress-io/cypress/issues/3817) I found referencing this kind of issue, but most suggestions included using `cy.wait()` 😒